### PR TITLE
TensorFlow Lattice notebooks

### PR DESCRIPTION
**Pull request by si-you**
These notebook contains TensorFlow Lattice examples.

### CODE REVIEW

File: notebooks/tensorflow_lattice/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+## Mac instructions
+
+First, we'll setup the virtual environment and install software as follows.
+
+```bash
+$ sudo easy_install pip
+$ cd ~/
+~$ virtuaenv --system-site-packages tf-lattice
```
Comment (random-forests) [inline notebooks/tensorflow_lattice/README.md]: s/virtuaenv/virtualenv

File: notebooks/tensorflow_lattice/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+*   MNIST dataset
+
+## Linux instructions
+
+First, we'll setup the virtual environment and install software as follows.
+
+```bash
+$ sudo apt-get install python-pip python-dev python-virtualenv
+$ cd ~/
+~$ virtuaenv --system-site-packages tf-lattice
```
Comment (random-forests) [inline notebooks/tensorflow_lattice/README.md]: s/vietuaenv/virtualenv

File: notebooks/tensorflow_lattice/01_calibrator_basics.ipynb [hunk 3]
```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basics of 1d calibrators\n",
+    "In this note book, we'll explain one dimensional calibrators."
```
Comment (random-forests) [inline notebooks/tensorflow_lattice/01_calibrator_basics.ipynb]: s/note book/notebook

**Comment by si-you** [comment]
Done! Please take a look!

**Pull request merged**